.github/actions/build-deps/action.yml

```
@@ -33,11 +38,13 @@ runs:
         BUILD_OPTION: ${{ inputs.force_build == 'true' && '*' || 'missing' }}
         BUILD_TYPE: ${{ inputs.build_type }}
         LOG_VERBOSITY: ${{ inputs.log_verbosity }}
+        CMAKE_SANITIZERS: ${{ inputs.sanitizers }}
```

```suggestion
        XRPLF_SANITIZERS: ${{ inputs.sanitizers }}
```

It's not purely CMake option or standard CMake option name, so I think it would be better to not have `cmake` in the name

---

Yeah I named it such because it is a compile time option and other such options are named with cmake prefix.

---

Could you please give an example of another user-defined option which we start with `CMAKE_`?

---

`cmake_args`,  `cmake_target` in other yml files.

---

Ok, but these are still only passed to cmake, and not to sth-else (like conan in this example)

---

There is already an option like this `BUILD_TYPE` - we pass it both to Conan and CMake, so I think the best choices are - `XRPLF_SANITIZERS` and just `SANITIZERS` as well

---

Going with `SANITIZERS`.